File: Cuadrado.java
package rectangulo;

class Cuadrado extends Rectangulo implements Comparable<Cuadrado> {
  
  
  
  Cuadrado(int lado) {
    super(lado, lado); 
  }
/**
 * compara un cuadrado con este
 * @return int 
 */
  public int compareTo(Cuadrado cuadrado) {
    return this.compareTo(cuadrado);
  }
/**
 * devuelve el lado del cuadrado  
 * @return lado int
 */
  public int getLado() {
    return this.getAlto();
  }
/**
 * establece el valor del lado del cuadrado
 * @param lado int
 */
  public void setLado(int lado) {
    this.setAlto(lado);
    this.setAncho(lado);
  }
@Override
public String toString() {
  return "Cuadrado [getLado()=" + getLado() + "]";
}
  
  
}

File: Cuadrado.py
'''
@author: Alberto Miguel Martinez Jimenez
'''
from Rectangulo import Rectangulo

class Cuadrado(Rectangulo):
    def __init__(self,lado):
        super().__init__(lado,lado)
      
if __name__ == "__main__":
    C=Cuadrado(5)
    C.pintarRectangulo()
  
